Cargo.toml

```diff
@@ -112,10 +112,11 @@ tokio-util = "0.7.8"
 url = "2.4.0"
 
 [workspace.dependencies.arrow2]
-# branch = "daft-fork"
+# TODO: Update this to daft-fork
+# branch = "jay/fd-add-rename-test"
```

Comment:
Change was added here to populate field_ids in the inferred arrow2 Field's metadata

Reply:
Once this PR is approved, I'll update this to point to a rebased `daft-fork` again